lib/components/ActivityIndicator.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <label v-show="busy">
+        <slot></slot>
+    </label>
+</template>
+
+<script>
+export default {
+    name: 'ActivityIndicator',
+    props: [
+        'busy',
+        'busyChange', // an event.
```

Comment:
this prop is redundant I think, @mylmz10 please confirm

Reply:
yes, it's just "busy" prop with type, default etc

Reply:
@mylmz10 shouldn't we remove it then?